fsb/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "fsb_rust"
```

Comment:
RustCrypto owns the `fsb` name. You can use that.

@newpavlov can you add `rustcrypto:hashes` as an owner?